yarn-project/pxe/src/contract_function_simulator/contract_function_simulator.ts

```diff
@@ -82,7 +82,7 @@ export class ContractFunctionSimulator {
     const txRequestHash = await request.toTxRequest().hash();
     const noteCache = new ExecutionNoteCache(txRequestHash);
 
-    const context = new PrivateExecutionOracle(
+    const privateExecutionOracle = new PrivateExecutionOracle(
```

Comment:
Unrelated change --> this naming was stale from the times we had `ClientExecutionContext`.